shortReadsqc.wdl

```diff
@@ -218,16 +248,16 @@ task rqcfilter {
             File stat = filename_stat
             File stat2 = filename_stat2
             File info_file = filename_reproduce
-            File filtered = glob("filtered/*fastq.gz")[0]
+            File filtered = glob("filtered/*.anqdpht.fastq.gz")[0]
```

Comment:
remove glob for `File filtered = "filtered/raw.anqdpht.fastq.gz"`
per: https://github.com/microbiomedata/metaT_ReadsQC/pull/8#discussion_r1720178665

Reply:
Jaws test: 84451

Reply:
looks good